doc/conf.py

```diff
@@ -1382,6 +1382,7 @@ def reset_warnings(gallery_conf, fname):
         r"iteritems is deprecated.*Use \.items instead\.",
         "is_categorical_dtype is deprecated.*",
         "The default of observed=False.*",
+        "When grouping with a length-1 list-like.*",
```

Comment:
Unrelated fix for https://app.circleci.com/pipelines/github/mne-tools/mne-python/22758/workflows/5b6b3106-2ca4-4ad2-8e68-0d63310cf410/jobs/63327